<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntAppealDlg</class>
 <widget class="QDialog" name="EntAppealDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3660</width>
    <height>2960</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MessageTipDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="FilletWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>280</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>3500</width>
       <height>2800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:4px;
background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="labelTilte">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>160</y>
        <width>340</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12px &quot;微软雅黑&quot;;
color:#1F8BED;</string>
      </property>
      <property name="text">
       <string>请上传8M以内，JPG/PNG格式的营业执照图片</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="BtnWidget1" native="true">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>190</y>
        <width>330</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>80</height>
       </size>
      </property>
      <widget class="QPushButton" name="buttonOK">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>120</width>
         <height>40</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>72</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="clostPushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>32</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-image: url(:/ZcloudCommon/image/browser_close_nor.png);}
QPushButton:hover,pressed{
border-image: url(:/ZcloudCommon/image/browser_close_hov.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>上传营业执照</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLicense">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>30</y>
        <width>340</width>
        <height>150</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="labelUpload">
      <property name="geometry">
       <rect>
        <x>133</x>
        <y>30</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:18px 'Microsoft Yahei';</string>
      </property>
      <property name="text">
       <string>上传图片</string>
      </property>
     </widget>
     <zorder>labelLicense</zorder>
     <zorder>labelTilte</zorder>
     <zorder>BtnWidget1</zorder>
     <zorder>clostPushButton</zorder>
     <zorder>label</zorder>
     <zorder>labelUpload</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EntCenterWidget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clostPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EntAppealDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
